src/clj/rems/application/commands.clj

```diff
@@ -66,6 +66,11 @@
 (s/defschema DecideCommand
   (assoc CommandWithComment
          :decision (s/enum :approved :rejected)))
+(s/defschema InviteActorCommand
+  (assoc CommandBase
+         :invitee {:name s/Str ; TODO figure out a better name. :actor would conflict with CommandBase
+                   :email s/Str}
```

Comment:
naming is hard!

Reply:
our commands so far use non-namespaced keys

Reply:
But that is not a strict limit? More like old thinking wrt. namespaced keywords and their ADR?

Reply:
Yeah, I was just thinking about API consistency

Reply:
This got solved by having separate InviteReviewer and InviteDecider commands